docs/6.Events.md

```diff
@@ -169,26 +177,36 @@ A worker strategy may ask the worker to dispatch events.
 From the ProcessQueueStrategy
 
 ```php
-public function onJobPop(WorkerEvent $e)
+public function onJobPop(ProcessQueueEvent $processQueueEvent)
 {
-    $queue   = $e->getQueue();
-    $options = $e->getOptions();
-    $job     = $queue->pop($options);
+    /** @var AbstractWorker $worker */
+    $worker       = $processQueueEvent->getTarget();
```

Comment:
Dedicated `getWorker` event? I never like the `getTarget` especially because I never know what the target is.